src/components/CheckBox.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import clsx from 'clsx'
+import { useFormContext } from 'react-hook-form'
+
+import './CheckBox.css'
+
+type Props = {
+  name: string
+  label: string
+  boxClassName?: string
+  labelClassName?: string
+  required?: boolean
+  wrapperClassName?: string
+}
+
+export default function CheckBox({ name, label, boxClassName, labelClassName, wrapperClassName, required }: Props) {
+  const { register, errors, trigger, formState } = useFormContext()
+
+  const triggerValidation = () => {
+    // eslint-disable-next-line
+    // @ts-ignore
+    formState.isDirty && trigger(name)
```

Comment:
C'est quoi l'erreur TS ici du coup ?

Reply:
Ça fait une erreur Argument of type 'string' is not assignable to parameter of type 'never[]'.